doc/source/pandas_supported.rst

```diff
@@ -0,0 +1,743 @@
+Pandas on Ray Supported Methods
+===============================
+
+For your convenience, we have compiled a list of currently implemented APIs and
+methods available in **Pandas on Ray**.
```

Comment:
Might be worth adding a note here pointing people to the documentation for installing the latest wheel (since this documentation will be out of sync with releases).

Reply:
Yeah that's a good idea. How often do you build the documentation?

Reply:
Hmm, well ideally it would happen automatically with each commit (it's probably possible to configure it this way). But in practice, whenever I push changes to the documentation I just manually rebuild it (and whenever someone else asks me to). I can also add you to the ReadTheDocs account so you can do it as well.